.eslintrc.js

```diff
@@ -20,5 +20,6 @@ module.exports = {
     '@typescript-eslint/interface-name-prefix': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-unused-vars': 'off', // False positives from decorators
```

Comment:
would it be more safe to add ignore specifically to such places instead of switching off the check?

Reply:
Looks like this was no longer needed